packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts

```diff
@@ -2343,8 +2337,9 @@ export async function calculateTransferFees(asset: NftAsset, signer: Signer, rec
   } else {
     tokenContract = new ethers.Contract(asset.asset_contract.address, ERC1155_ABI, signer)
     args.push('1')
+    args.push('0x')
   }
-  return safeGasEstimation(tokenContract.gasEstimation.safeTransferFrom, args, { gasLimit: 250_00 })
+  return safeGasEstimation(tokenContract.estimateGas.safeTransferFrom, args, { gasLimit: 250_000 })
```

Comment:
should the `gasLimit` be hardcoded?  should it maybe be from ETH fees API, they could add a new one for NFTs such they did for contract calls..?

Reply:
It's a fallback